package.json

```diff
@@ -45,32 +45,30 @@
     "map-stream": "*",
     "replace-ext": "*",
     "require-dot-file": "*",
-    "run-sequence": "*",
     "yamljs": "*"
   },
   "dependencies": {
     "better-console": "*",
     "del": "^3.0.0",
     "extend": "^3.0.2",
-    "gulp": "^3.9.1",
+    "gulp": "^4.0.0",
     "gulp-autoprefixer": "^6.0.0",
     "gulp-chmod": "^2.0.0",
+    "gulp-clean-css": "^3.10.0",
     "gulp-clone": "^2.0.1",
     "gulp-concat": "^2.6.1",
     "gulp-concat-css": "^3.1.0",
     "gulp-copy": "4.0.1",
     "gulp-dedupe": "0.0.2",
     "gulp-flatten": "^0.4.0",
     "gulp-header": "^2.0.5",
-    "gulp-help": "^1.6.1",
+    "gulp-help": "github:chmontgomery/gulp-help#gulp4",
```

Comment:
I'm just reviewing these changes to merge into my own fork and saw this.  Given that this PR is intended to solve security warnings, I don't think we can trust adding a dependency on a floating github branch.  It could be modified at any time, making builds unrepeatable and adding an attack vector.

Is there no published version that can be used?

Reply:
There is never going to be a gulp4 release of `gulp-help` https://github.com/chmontgomery/gulp-help/issues/31#issuecomment-449979322

Reply:
How about referencing the commit hash of the HEAD of that branch?

This should work (never tried it though):

github:chmontgomery/gulp-help#194e80e0545ff6af5d10eeba7e224d0da71eb8d3

Reply:
@badgerwithagun You could also check out the Fomantic-UI fork at https://github.com/fomantic/Fomantic-UI . We already have updated to gulp 4 and gulp-help will be removed with the next release